core/templates/domain/user/user-profile.model.ts

```diff
@@ -27,7 +27,9 @@ export interface UserProfileBackendDict {
   'username_of_viewed_profile': string;
   'user_bio': string;
   'subject_interests': string[];
-  'first_contribution_msec': number;
+  // The time in milliseconds when the user first contributed to Oppia.
+  // This property is initially set as null for a new user .
+  'first_contribution_msec': number | null;
```

Comment:
```suggestion
  'first_contribution_msec'!: number;
```

Reply:
`null` is assigned as a value later so it used a  type, `!` is for bypassing `strictPropertyInitialization` error, we need not to use it here because it's a model file and we don't initialize properties here.